### DISCUSSION TITLE
samples/drivers/ipm/ipm_esp32 does not compile

### DISCUSSION THREAD

**Discussion by rhvarrier** [zephyrproject-rtos/zephyr#80483] [Q&A]
I'm trying to test ESP32 IPM example of an ESP32S3 DevkitM however I'm getting the following error:

```
cc1: fatal error: /workspaces/esp32s3/build/ipm_esp32_appcpu-prefix/src/ipm_esp32_appcpu-build/zephyr/esp32_appcpu_firmware.c: No such file or directory
```

I'm using the following command to build:
```
west build -b esp32s3_devkitm/esp32s3/procpu samples/drivers/ipm/ipm_esp32
```
And I using Zephyr SDK version 0.16.8

Has anyone encountered this problem ?

**Comment by enthunilu** [comment]
Hi,

I am also getting the same issue as you mentioned.
I am using Zephyr SDK v0.16.5.

In case I get any hint or solution, I'll post here.

**Reply by rhvarrier** [reply]
Maybe @sylvioalves, do you have any idea about this.

**Comment by rftafas** [comment]
If possible, try this again using latest (head after 4.0.0). There were fixes to IPM.
